openlibrary/macros/SearchResultsWork.html

```diff
@@ -1,45 +1,53 @@
-$def with (results, fulltext=False)
+$def with (doc)
 
-$ _ = i18n.get_namespace('/search')
+<li class="searchResultItem" itemscope itemtype="https://schema.org/Book">
+  <span class="bookcover">
+    $if doc.get('cover_i'):
+        $ cover =  get_coverstore_url() + "/b/id/%s-S.jpg" % doc.cover_i
+    $elif doc.get('cover_edition_key'):
+        $ cover = get_coverstore_url() + "/b/olid/%s-S.jpg" % doc.cover_edition_key
+    $elif doc.ia:
+        $ cover = "//archive.org/download/%s/page/cover_w60_h60.jpg" % doc.ia[0]
+    $else:
+        $ cover = "/images/icons/avatar_book-sm.png"
+    <a href="$(doc.get('url', doc.key + '/' + doc.title))"><img itemprop="image" src="$cover" height="70" alt="Cover of: $doc.title$(': ' + doc.subtitle if doc.get('subtitle', None) else '')" title="Cover of: $doc.title$(': ' + doc.subtitle if doc.get('subtitle', None) else '')"/></a>
```

Comment:
in all examples it appears to be. We could remove it and have the router add it correctly via a redirect, but it seems to make more sense to get the seo benefit of including it.